spec/user_spec.rb

```diff
@@ -0,0 +1,24 @@
+describe User do
+  describe '.create' do
+    it 'creates a new user' do
+      user = User.create(peeper: 'Lupita', username: 'Lupi', email: 'lupitareyes@example.es', password: 'lupitrona')
+
+      expect(user.id).to eq 3
```

Comment:
This feels brittle (it will break easily) because you're testing for a specific ID.  Instead you could do something like
```ruby
expect { User.create(params) }.to change { User.all.count }.by 1
```

This would test that something is entered into the DB and not be effected by what's already in the DB